modules/javafx.base/src/main/java/javafx/beans/value/ObservableValue.java

```diff
@@ -137,4 +139,55 @@
      * @return The current value
      */
     T getValue();
+
+    /**
+     * Returns an {@link ObservableValue} which provides a mapping of the value
+     * held by this {@code ObservableValue}, and provides {@code null} when this
+     * {@code ObservableValue} holds {@code null}.
+     *
+     * @param <U> the type of values held by the resulting {@code ObservableValue}
+     * @param mapper a {@link Function} which converts a given value to a new value, cannot be null
+     * @return an {@link ObservableValue} which provides a mapping of the value
+     *     held by this {@code ObservableValue}, and provides {@code null} when
+     *     this {@code ObservableValue} holds {@code null}, never null
```

Comment:
Good idea to mention that it can hold `null`.
I slightly prefer to say that the returned `ObservableValue` holds the result of the mapping rather than holds the mapping. I don't really mind it, but it's the phrasing used in the method description "holds the result of applying a mapping". "The mapping" itself could be mistaken for the mapping `Function` in my opinion. If you think it's clear, you can change it to that phrasing, it's also fine.